File: helpers/randomGame.js
const fs = require('fs');

var game = function game(){
  var textByLine = fs.readFileSync("./games.txt").toString().split("\n");
  
  var gameList = [];
  for(var i = 0; i < textByLine.length; i++) {
    gameList.push(textByLine[i].trim());
  }

  var randomIndex = Math.floor(Math.random() * gameList.length);
  return gameList[randomIndex];
}

var addGame = function addGame(input){
  fs.appendFile('./games.txt', '\n' + input, (err) => {
    if (err) throw err;
  });
}

module.exports.game = game;
module.exports.addGame = addGame;

File: helpers/randomGame.py
import random

def randomGame():
    gamesByLine = []
    with open('games.txt', 'r') as f:
        gamesByLine = f.readlines()

    return gamesByLine[random.randint(0, len(gamesByLine) - 1)][:-1]

def addGame(game):
    with open('games.txt', 'a') as f:
        f.write('\n' + game)